.github/workflows/ci.yml

```diff
@@ -216,8 +263,10 @@ jobs:
         -DCMAKE_INSTALL_PREFIX:PATH=../install
         -DF3D_STRICT_BUILD=ON
         -DF3D_MODULE_OCCT=ON
+        -DF3D_MODULE_ASSIMP=ON
         -DCMAKE_BUILD_TYPE=Release
         -DVTK_DIR:PATH=../dependencies/vtk_install/lib/cmake/vtk
+        -Dassimp_DIR:PATH=$(pwd)/../dependencies/assimp_install/lib/cmake/assimp-5.1
```

Comment:
pwd needed ? this was needed for opencascade for an undetermined reason

Reply:
Yes, it is required here as well. I have no idea why

Reply:
mentions assimp in the comment on top, and add a TODO if you want to investigage

Reply:
add assimp in the comment above